blocks/header-nav-chain-block/chains/header-nav-chain-block/default.jsx

```diff
@@ -312,6 +386,18 @@ Nav.propTypes = {
       label: 'Horizontal Links hierarchy',
       group: 'Configure content',
     }),
+    desktopNavivationStartHeight: PropTypes.number.tag({
+      label: 'Starting desktop navigation bar height',
+      group: 'Logo',
+      max: 200,
+      min: 56,
+      defaultValue: 56,
+    }),
+    shrinkDesktopNavivationHeight: PropTypes.number.tag({
+      label: 'Shrink navigation bar after scrolling ',
+      group: 'Logo',
+      min: 56,
```

Comment:
Should this also have a `max` of `200`? Fusion doesn't seem be honoring the `max` at the moment but that seems to be a separate Fusion bug.

Reply:
Yeah, I have the same thought like you before. But when thinking about it again, then at some point we can configure scroll position more than 200px then animation happening